client/src/lib/slate-pointers/PointerExportMark.tsx

```diff
@@ -92,24 +35,57 @@ export class PointerExportMark extends React.Component<any, any> {
       availablePointers,
       nodeAsJson,
       children,
-      isDisplayMode
     }: any = this.props;
 
+    const pointerExportBackground: any =
+      isSelected
+      ?
+      "rgba(85, 228, 38, 0.9)"
+      :
+      "rgba(200, 243, 197, 0.5)";
+
     const { pointerIndex }: any = propsToPointerDetails({
       blockEditor,
       availablePointers,
       nodeAsJson
     });
+
+    const styles = StyleSheet.create({
+      OuterPointerExportStyle: {
+        ":before": {
+          backgroundColor: "rgba(12, 165, 0, 0.63)",
+          color: "rgb(233, 239, 233)",
+          content: `"$${parseInt(pointerIndex, 10) + 1}"`,
+          borderRadius: "4px 0px 0px 4px",
+          padding: "0px 3px",
+        },
+      },
+
+      PointerExportStyle: {
+        background: `${pointerExportBackground}`,
+        color: "#000000",
+        marginLeft: "0.5px",
+        transition: "background 0.2s",
+
+        ":before": {
+          color: "rgba(12, 165, 0, 0.63)",
+          content: `"["`,
+          font: "800 1.2em sans-serif",
+        },
+
+        ":after": {
+          color: "rgba(12, 165, 0, 0.63)",
+          content: `"]"`,
+          font: "800 1.2em sans-serif",
+        },
+      },
+    });
```

Comment:
Happy that this :before / :after use works. I would appreciate it if this could be DRYed up a bit. I'm also not totally sure this can't be done with styled-components, with a bit of trickery, I may mess with that later.